.vscode/settings.json

```diff
@@ -3,5 +3,8 @@
     "editor.formatOnSave": true,
     "[typescript]": {
         "editor.defaultFormatter": "esbenp.prettier-vscode"
+    },
+    "[html]": {
+        "editor.defaultFormatter": "esbenp.prettier-vscode"
```

Comment:
This change is intended to be committed?

Reply:
yes